virgo/ui/pages/canvas_page.py

```diff
@@ -12,25 +12,25 @@
 if TYPE_CHECKING:
     from virgo.app import App
 
-#TODO: turn this into a class and refactor
```

Comment:
Lets keep all the #TODO comments for now, I'll put them in a list and remove later.